greenboot-health-check.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ -x /usr/local/libexec/yggdrasil/device-worker ]; then
```

Comment:
Can we check the status of the yggdrasild, nft and podman services which are required for the k4e-agent to operate ?

Reply:
I added a relevant task to enrich all 4 parts of greenboot checks 
https://issues.redhat.com/browse/ECOPROJECT-340